src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Imaging/WriteableBitmap.cs

```diff
@@ -1085,7 +1085,7 @@ internal override void FinalizeCreation()
 
             if (sourceBuffer.Rank == 1)
             {
-                if (sourceBuffer.GetLength(0) <= 0)
+                if (sourceBuffer.Length == 0)
```

Comment:
For one dimensional arrays I would prefer to use "Length"
but would like to encourage "GetLength" for multidimensional arrays for consistency reasons.

Reply:
We can do it for consistency reasons, I'll push it later today. I'll cache the result of `GetLength` as it is queried twice though.

Reply:
@hiteshwpfmsft Thanks, I've also changed the `<=` compares to `==` there as it makes no sense to compare less than while at it.

Reply:
Agree!